WDL/CLI.py

```diff
@@ -375,21 +402,30 @@ def cromwell(uri, inputs, json_only, empty, check_quant, rundir=None, path=None,
     with open(cromwell_options_filename, "w") as options_json:
         print(json.dumps(cromwell_options, indent=2), file=options_json)
 
+    # setup Cromwell config file
+    config_setting = None
+    if config:
+        config_setting = "-Dconfig.file={}".format(config)
+    elif "CROMWELL_CONFIG" in os.environ:
+        config_setting = "-Dconfig.file={}".format(os.getenv("CROMWELL_CONFIG"))
+
     # launch Cromwell
-    jarpath = ensure_cromwell_jar()
-    cromwell_cmd = [
-        "java",
-        "-DLOG_LEVEL=warn",
-        "-DLOG_MODE=pretty",
-        "-jar",
-        jarpath,
-        "run",
-        (os.path.abspath(uri) if "://" not in uri else uri),
-        "-o",
-        cromwell_options_filename,
-        "-i",
-        input_json_filename,
-    ]
+    jarpath = ensure_cromwell_jar(jarfile)
+    cromwell_cmd = ["java", "-DLOG_LEVEL=warn", "-DLOG_LEVEL=pretty"]
```

Comment:
It looks like for the third element here I scribbled 8-DLOG_LEVEL=pretty` instead of `-DLOG_MODE=pretty` -- probably this caused the log blowup